### PR TITLE
Task/schedule doc

### PR DESCRIPTION
**Pull request by MikePetitfils**

### CODE REVIEW

File: schedule/@schedule.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+layout: main
+title: Schedule
+nav_order: 2
```
Comment (EvenModiguy) [inline schedule/@schedule.md:4]: use this instead:
```suggestion
nav_order: 5
```

File: schedule/ios/1.0.0/communicating_with.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+permalink: /schedule/ios/communicating-with
+---
+
+# Communicating with
+
+---
+
+## Other SDK
+
+---
```
Comment (EvenModiguy) [inline schedule/ios/1.0.0/communicating_with.md:17]: Please remove this file if there is no documentation.

File: schedule/ios/@ios.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+# Schedule iOS
+
+<!-- ## Changelog & Releases
+
+| Version |  |
+| --- | --- |
+| 0.2.0 |  <label class="label label-green">Latest</label>  |
+
+--- -->
```
Comment (EvenModiguy) [inline schedule/ios/@ios.md:18]: Please remove those comments.

File: schedule/ios/1.0.0/getting_started.md [hunk 4]
```diff
@@ -0,0 +1,208 @@
+---
+layout: main
+title: Getting started
+parent: Schedule iOS
+grand_parent: Schedule
+nav_order: 2
```
Comment (EvenModiguy) [inline schedule/ios/1.0.0/getting_started.md:6]: Use this instead.
```suggestion
nav_order: 1
```

File: schedule/ios/1.0.0/screens.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
+---
+layout: main
+title: Screens
+parent: Schedule iOS
+grand_parent: Schedule
+nav_order: 3
```
Comment (EvenModiguy) [inline schedule/ios/1.0.0/screens.md:6]: Use this instead
```suggestion
nav_order: 2
```

**Review by EvenModiguy — CHANGES_REQUESTED** [review]

**Review by RachikAbidi — APPROVED** [review]

**Pull request merged**